azext_iot/_params.py

```diff
@@ -815,6 +815,33 @@ def load_arguments(self, _):
             arg_type=auth_type_dataplane_param_type,
         )
 
+    with self.argument_context("iot edge module image terms") as context:
+        context.argument(
+            "offerId",
+            options_list=["--offer", "-f"],
+            help="IoT Edge module image offer.",
+        )
+        context.argument(
+            "planId",
+            options_list=["--plan"],
+            help="IoT Edge module image billing plan.",
+        )
+        context.argument(
+            "publisherId",
+            options_list=["--publisher", "-p"],
+            help="IoT Edge module image publisher.",
+        )
+        context.argument(
```

Comment:
urn should be removed unless a provider API for lookups is discovered or created.